lib/pegboard/container-runner/src/main.rs

```diff
@@ -33,6 +33,12 @@ fn main() -> anyhow::Result<()> {
 		.context("`container_dir` arg required")?;
 	let pegboard_container_dir = Path::new(&pegboard_container_dir);
 
+	// Write PID to file
+	fs::write(
+		pegboard_container_dir.join("pid"),
```

Comment:
nit: use consts for all files that are written. makes it easy to know what's in the folder

Reply:
could you elaborate?